docs/docsite/rst/porting_guides/porting_guide_4.rst

```diff
@@ -266,9 +291,9 @@ Deprecated Features
 ansible.windows
 ~~~~~~~~~~~~~~~
 
-- win_updates - Deprecated the ``filtered_reason`` return value for each filtered up in favor of ``filtered_reasons``. This has been done to show all the reasons why an update was filtered and not just the first reason.
+- win_updates - Deprecated the ``filtered_reason`` return value for each filtered up in favour of ``filtered_reasons``. This has been done to show all the reasons why an update was filtered and not just the first reason.
 - win_updates - Deprecated the ``use_scheduled_task`` option as it is no longer used.
-- win_updates - Deprecated the ``whitelist`` and ``blacklist`` options in favor of ``accept_list`` and ``reject_list`` respectively to conform to the new standards used in Ansible for these types of options.
+- win_updates - Deprecated the ``whitelist`` and ``blacklist`` options in favour of ``accept_list`` and ``reject_list`` respectively to conform to the new standards used in Ansible for these types of options.
```

Comment:
@abadger, The style guide [recommends using American English](https://docs.ansible.com/ansible/latest/dev_guide/style_guide/index.html#stylistic-cheat-sheet),  so "favor" is the approved spelling here.

Reply:
Hey @ariordan-redhat We won't be able to change that here.  It needs to be changed in the collection that contains win_updates ( https://github.com/ansible-collections/ansible.windows ).  Once/if it is changed there, then the change will be reflected here the next time we build this file.

Reply:
Thanks, @abadger .